dask_glm/algorithms.py

```diff
@@ -272,8 +267,9 @@ def compute_loss_grad(beta, X, y):
     return beta
 
 
+@normalize
 def proximal_grad(X, y, regularizer='l1', lamduh=0.1, family=Logistic,
-                  max_iter=100, tol=1e-8, verbose=False):
+                  max_iter=100, tol=1e-8):
```

Comment:
Yea, actually all the algorithms should; it won't break anything now but should definitely be updated in whatever PR gets merged next.